schemas/input-feed.json

```diff
@@ -0,0 +1,46 @@
+[
+  {
+    "CRE_ID": "<ID in here>",
+    "links": {
```

Comment:
This tries to both define a CRE and link a CRE with multiple other things,
maybe it's better if this is split into two.
A definiton such as:
` {"CRE":{"description":<string>,"id":<fromatted string>,"tags":[<string>], "requirements":[<string>]}}`
and a link such as 
`{"CRE_LINK":{"CRE_ID":<formated_str>,"name":<ASVS,ISO,PCI,etc>,"id":<ASVS_ID,3.3.1a,etc>,"tags":[<a>,<list>,<of>,<free>,<text>,<tags>]}
`
The link can then be something like:

`{"CRE_LINK":{"CRE_ID":<formated_str>,"name":"ASVS", "id":"ASVS_ID","tags":[]}
`
or

`{"CRE_LINK":{"CRE_ID":<formated_str>,"name":"WSTG", "id":"4.3.2.1","tags":["tests"]}
`

Reply:
This document pertains to people that are pushing updates to the CREs, and not creating CREs. CRE metadata is not required in our case. This is more focused to let a project to integrate their IDs with the CREs. The links in the object are links to ASVS-ID, WSTG-ID, etc., and not the CRE.

Reply:
i see, so what do you think about the second part of the suggestion(one "CRE_LINK" per json object with tags, instead of one object with an array of CRE links each under some category?